### PR TITLE
Rephrase texts for fsensor detection and cleanup

### PR DESCRIPTION
**Pull request by DRracer**

### CODE REVIEW

**Review by XPila — APPROVED** [review]

File: Firmware/Marlin_main.cpp [hunk 1]
```diff
@@ -9484,10 +9484,10 @@ static uint8_t nFSCheckCount=0;
                          if(nFSCheckCount>FS_CHECK_COUNT)
                          {
                               nFSCheckCount=0;    // not necessary
-                              oFsensorPCB=ClFsensorPCB::_Rev03b;
+                              oFsensorPCB=ClFsensorPCB::_Rev04;
                               eeprom_update_byte((uint8_t*)EEPROM_FSENSOR_PCB,(uint8_t)oFsensorPCB);
                               printf_IRSensorAnalogBoardChange(true);
-                              lcd_setstatuspgm(_i("FS rev. 03b or newer"));
+                              lcd_setstatuspgm(_i("FS v0.4 or newer"));
```
Comment (3d-gussner) [inline Firmware/Marlin_main.cpp:9490]: Why not using `MSG_04_OR_NEWER` ?

File: Firmware/ultralcd.cpp [hunk 2]
```diff
@@ -2166,14 +2166,14 @@ static void lcd_support_menu()
   switch(oFsensorPCB)
        {
        case ClFsensorPCB::_Old:
-            MENU_ITEM_BACK_P(PSTR(" 03 or older"));
+            MENU_ITEM_BACK_P(MSG_03_OR_OLDER);
```
Comment (ghost) [inline Firmware/ultralcd.cpp:2169]: MENU_ITEM_BACK_P(_T(MSG_03_OR_OLDER));

File: Firmware/ultralcd.cpp [hunk 3]
```diff
@@ -2166,14 +2166,14 @@ static void lcd_support_menu()
...
             break;
-       case ClFsensorPCB::_Rev03b:
-            MENU_ITEM_BACK_P(PSTR(" 03b or newer"));
+       case ClFsensorPCB::_Rev04:
+            MENU_ITEM_BACK_P(MSG_04_OR_NEWER);
```
Comment (ghost) [inline Firmware/ultralcd.cpp:2172]: MENU_ITEM_BACK_P(_T(MSG_04_OR_NEWER));

Comment (DRracer) [reply Firmware/ultralcd.cpp:2172]: @ovariludovit thanks, you are right

**Pull request merged**